tests/store/test_footprints.py

```diff
@@ -277,7 +273,7 @@ def test_read_footprint_high_spatial_resolution():
         ),
     ],
 )
-def test_read_footprint_co2(site, inlet, metmodel, start, end, filename):
+def test_read_footprint_co2(site, inlet, metmodel, start, end, filename, standardise):
```

Comment:
Same small comment as for `test_read_footprint_high_spatial_resolution` - would be nice to keep the variable definition and layout consistent.